tests/acceptance/features/webUIUserKeysType/userKeys.feature

```diff
@@ -0,0 +1,27 @@
+@webUI @skipOnEncryptionType:masterkey
+Feature: encrypt files using user specific keys
+  As an admin
+  I want to be able to encrypt user files using user specific keys
+  So that users can use specific keys for encrypting their files
+
+  Background:
+    Given user "brand-new-user" has been created
+    And the app "encryption" has been enabled
+    And encryption has been enabled
+
+  Scenario: encrypt using user keys based encryption
+    When the administrator sets the encryption type to "user-keys" using the occ command
+    And the administrator encrypts all data using the occ command
+    Then the file "textfile0.txt" of user "brand-new-user" should be encrypted
+
```

Comment:
one extra test that is a user logs in after encryption enabled the data should be encrypted